src/PrestaShopBundle/Resources/config/services.yml

```diff
@@ -7,6 +7,9 @@ parameters:
     prestashop.security.voter.product.class: PrestaShopBundle\Security\Voter\PageVoter
 
 services:
+    _defaults:
+        public: true
+
```

Comment:
So you had to set it as default AND for each service?

Reply:
only for them where Symfony cry a river, but it was not supposed to happen "in the book" :p